share/instruments/instruments.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <museScore>
       <Genre id="common">
-            <name>Common instruments</name>
+            <name>Common</name>
       </Genre>
       <Genre id="jazz">
-            <name>Jazz instruments</name>
+            <name>Jazz</name>
       </Genre>
       <Genre id="orchestra">
-            <name>Orchestral instruments</name>
+            <name>Orchestral</name>
       </Genre>
       <Genre id="earlymusic">
             <name>Early music</name>
       </Genre>
-      <Genre id="ethnic">
-            <name>Ethnic instruments</name>
+      <Genre id="world">
+            <name>World music</name>
```

Comment:
We could change the name to `World Music while keeping the it at `ethnic`, would make the change much leaner

Reply:
I'm still of the opinion that this is just too much of "political correctness" though, "Ethnic" does not have a bad connotation to me and "World music" doesn't say anything to me, also it does not match the single word term of all the other genres got, so I'd rather keep "Ethnic".